Gemfile

```diff
@@ -33,6 +33,8 @@ group :development, :test do
 end
 
 group :development do
+  gem 'better_errors'
```

Comment:
No I'm fine with leaving them. Just making sure they were being added for good reasons.